index.src.html

```diff
@@ -732,8 +730,6 @@
 --------------------------------------------------------------------------------------------
 Run these steps:
 1. If `state` is not `"configured"`, throw an {{InvalidStateError}}.
```

Comment:
Not strictly necessary, but re-numbering here helps reading the source, I find.

Reply:
Done. Fixed elsewhere too.